packages/expo-sqlite/CHANGELOG.md

```diff
@@ -4,10 +4,14 @@
 
 ### 🛠 Breaking changes
 
+- Changed behavior of `rowAfected` value in result of `executeSql` method in iOS to correctly handle cascade mutations and be on par with Android. ([@9137](https://github.com/expo/expo/pull/9317) by [@mczernek](https://github.com/mczernek))
```

Comment:
It's not a breaking change — `expo-sqlite` should comply with `WebSQL` specs so I think our implementation was broken in that place and no one should have relied on the current behavior of `rowsAffected`. If someone does rely on this, it's his fault and not our 😛